src/t_stream.c

```diff
@@ -1807,9 +1843,7 @@ void xaddCommand(client *c) {
 
     /* Append using the low level function and return the ID. */
     streamID id;
-    if (streamAppendItem(s,c->argv+field_pos,(c->argc-field_pos)/2,
-        &id, parsed_args.id_given ? &parsed_args.id : NULL) == C_ERR)
-    {
```

Comment:
any reason you modified 3 lines instead of one? :stuck_out_tongue:

Reply:
I followed your formatting rulez while I was at it :)

Reply:
my formatting (guidelines) are flexible, and are different for new code and existing code.
in this case the condition was already split into 3 lines, you made it longer by some 15 characters, so i don't see why you should take this opportunity to unify it into one (maybe if you had removed 30 chars from the condition it would have been right).
the effect of your change is:
1. line that is arguably too long
2. blame log destruction of 3 lines instead of 1.
i would suggest to change just the one.

Reply:
Resolved via 318c4868d